### PR TITLE
Add extra test for statement by guid removal method

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: tests/unit/Statement/StatementListTest.php [hunk 1]
```diff
@@ -260,7 +260,7 @@ public function testAddStatementWithGuid() {
 		$this->assertEquals( new StatementList( $statement ), $list );
 	}
 
-	public function testRemoveStatementsWithGuid_singleStatementRemoved() {
+	public function testGivenGuidOfPresenStatement_statementIsRemoved() {
```
Comment (thiemowmde) [inline tests/unit/Statement/StatementListTest.php]: "Presen"?

File: tests/unit/Statement/StatementListTest.php [hunk 2]
```diff
@@ -275,7 +275,7 @@ public function testRemoveStatementsWithGuid_singleStatementRemoved() {
 		$this->assertEquals( $statements, $list->toArray() );
 	}
 
-	public function testRemoveStatementsWithGuid_multipleStatementsRemoved() {
+	public function testGivenGuidOfMulitpleStatements_multipleStatementsAreRemoved() {
```
Comment (thiemowmde) [inline tests/unit/Statement/StatementListTest.php]: "Mulitple"? And merged? What type of code review is that?

Comment (Benestar) [inline tests/unit/Statement/StatementListTest.php]: Do you know about this? http://dan.hersam.com/2005/01/27/reading-jumbled-letters/

Comment (JeroenDeDauw) [inline tests/unit/Statement/StatementListTest.php]: @thiemowmde even if you spot such typos in a case similar to this one, IMO you are wasting everyone's time by -1'ing and not just merging and then fixing it yourself

**Pull request merged**